specification/indices/get/IndicesGetRequest.ts

```diff
@@ -64,5 +77,18 @@ export interface Request extends RequestBase {
      * @server_default 30s
      */
     master_timeout?: Time
+    /**
+     * Return only information on specified index features
+     * @server_default [aliases, mappings, settings]
```

Comment:
See https://github.com/elastic/elasticsearch-specification/blob/main/docs/modeling-guide.md#server_default:

```suggestion
     * @server_default ['aliases', 'mappings', 'settings']
```